### PR TITLE
Update device-swap description

### LINKED ISSUE #16: Include Device Swap description
Proposal to include in API description, avoiding explicit Multi-SIM additional discussion (only including it as new device IMEI is associated to same MSISDN). We can later follow the discussion on considering only new multiSIM devices or consider any change.

**Device swap**: A Device Swap is a process in which a user's mobile phone number (MSISDN) is associated with a new device (IMEI).

This typically occurs when the user switches to a new mobile device, either by upgrading, replacing a lost or damaged device, or other similar reasons. A Device Swap also happens during actions like activating a new device associated with the same phone number, also known as multiSIM. New Device activation is considered as a Device Swap as well, as the phone number is associated with a device it was not associated before.

_Originally posted by @jgarciahospital in https://github.com/camaraproject/DeviceSwap/discussions/11#discussioncomment-10726846_

### PR DESCRIPTION
**Pull request by jgarciahospital**
#### What type of PR is this?

* documentation

#### What this PR does / why we need it:

Update API description including detailed Device Swap behaviour.

#### Which issue(s) this PR fixes:

Fixes #16

### CODE REVIEW

**Review by bigludo7 — APPROVED** [review]
LGTM

**Comment by jgarciahospital** [comment]
Description updated as agreed with @StefanoFalsetto-CKHIOD

File: code/API_definitions/device-swap.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ info:
 
       # Relevant terms and definitions
 
-      **Device swap**: A Device Swap is a process in which a user's mobile phone number (MSISDN) is associated with a new device (IMEI).
+      **Device swap: A Device Swap is a process in which the association between a user's mobile phone number (MSISDN) and a device (IMEI) is created for the first time or changes for any reasons.
```
Comment (StefanoFalsetto-CKHIOD) [inline code/API_definitions/device-swap.yaml:20]: @jgarciahospital it seems you are not closing the bold sentence

Comment (jgarciahospital) [reply code/API_definitions/device-swap.yaml]: fixed! thanks

**Review by bigludo7 — APPROVED** [review]
LGTM

**Pull request merged**